mythril/laser/ethereum/instructions.py

```diff
@@ -68,10 +68,17 @@ def jumpdest_(self, global_state):
 
     @instruction
     def push_(self, global_state):
+        push_instruction = global_state.get_current_instruction()
+        push_value = push_instruction['argument'][2:]
         try:
-            value = BitVecVal(int(global_state.get_current_instruction()['argument'][2:], 16), 256)
+            no_of_bytes = int(push_instruction['opcode'][4:])
         except ValueError:
-            raise VmException('Invalid Argument for PUSH')
+            raise VmException('Invalid Push instruction')
```

Comment:
will this ever happen?

Reply:
nope, unless we want to input disassembled code in the future.